.github/workflows/render.yaml

```diff
@@ -0,0 +1,31 @@
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, labeled, reopened]
+
```

Comment:
Since these changes come from a fork I have executed the workflow there to demonstrate that it works

https://github.com/cameronraysmith/nature/actions/runs/9961835354